airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -777,6 +777,44 @@ definitions:
       type:
         type: string
         enum: [LegacyToPerPartitionStateMigration]
+  IncrementingCountCursor:
```

Comment:
@lmossman would you have a look at the YAML definition of this component to see if anything seems odd to you?